packages/eslint-config-mc-app/package.json

```diff
@@ -20,22 +20,18 @@
   },
   "main": "./index.js",
   "module": "./index.js",
-  "dependencies": {
-    "babel-eslint": "10.0.1",
-    "eslint": "5.12.0",
-    "eslint-config-airbnb-base": "13.1.0",
-    "eslint-config-prettier": "3.3.0",
-    "eslint-formatter-pretty": "2.0.0",
-    "eslint-plugin-babel": "5.3.0",
-    "eslint-plugin-cypress": "2.2.0",
-    "eslint-plugin-import": "2.14.0",
-    "eslint-plugin-jest": "22.1.2",
-    "eslint-plugin-jsx-a11y": "6.1.2",
-    "eslint-plugin-prefer-object-spread": "1.2.1",
-    "eslint-plugin-prettier": "3.0.1",
-    "eslint-plugin-react": "7.12.3"
-  },
+  "dependencies": {},
   "peerDependencies": {
-    "eslint": ">= 5.3"
+    "babel-eslint": "10.x",
```

Comment:
I found this way to determine a version range, I think it's simpler.

<img width="318" alt="screenshot 2019-01-13 at 16 53 55" src="https://user-images.githubusercontent.com/1110551/51087648-3a0bf800-1756-11e9-8e44-cf815114fc9a.png">

Reply:
We can change the other places where we define peer deps to use the same format.